cliquet/initialization.py

```diff
@@ -338,11 +338,14 @@ def initialize(config, version=None, project_name=None, default_settings=None):
         in application settings.
     :param dict default_settings: Override cliquet default settings values.
     """
-    if default_settings:
-        config.add_settings(default_settings)
-
     settings = config.get_settings()
 
+    if default_settings:
+        for key, value in default_settings.items():
+            if not settings.get(key):
```

Comment:
Yes indeed.

But we could still use ``cliquet.load_default_settings()`` in Kinto before loading Cliquet then.
Like suggested here http://cliquet.readthedocs.org/en/latest/ecosystem.html#configuration

Sorry, my brain looks like that currently:
![xp_defrag 3](https://cloud.githubusercontent.com/assets/546692/8453074/30b5078e-1ff1-11e5-85b2-f1e4cf7b1de5.png)